chanacceptor/chainedacceptor.go

```diff
@@ -61,11 +63,31 @@ func (c *ChainedAcceptor) Accept(req *ChannelAcceptRequest) *ChannelAcceptRespon
 		if acceptorResponse.RejectChannel() {
 			return acceptorResponse
 		}
+
+		// If we have accepted the channel, we need to set the other
+		// fields that were set in the response. However, since we are
+		// dealing with multiple responses, we need to make sure that we
+		// have not received inconsistent values (eg a csv delay of 1
+		// from one acceptor, and a delay of 120 from another). We
+		// set each value on our final response if it has not been set
+		// yet, and allow duplicate sets if the value is the same. If
+		// we cannot set a field, we return an error response.
+		var err error
+		finalResp, err = mergeResponse(finalResp, *acceptorResponse)
+		if err != nil {
+			log.Errorf("response for: %x has inconsistent values: %v",
+				req.OpenChanMsg.PendingChannelID, err)
+
+			return NewChannelAcceptResponse(
```

Comment:
note to self: if we ever do this in more than one place, define a specialized constructor